src/EnergyPlus/InputProcessing/InputProcessor.cc

```diff
@@ -573,7 +573,7 @@ bool InputProcessor::findDefault(Real64 &default_value, json const &schema_field
         auto const &default_val = find_default.value();
         if (default_val.is_string() && !default_val.get<std::string>().empty()) {
             // autosize and autocalculate
-            default_value = -99999;
+            default_value = DataGlobalConstants::AutoCalculate;
```

Comment:
Replace all literal `-99999` in InputProcessor.

Reply:
Great!